src/test/java/com/wikia/webdriver/testcases/contentreview/ContentReviewTestMode.java

```diff
@@ -0,0 +1,72 @@
+package com.wikia.webdriver.testcases.contentreview;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.editmode.VisualEditModePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialJsPage;
+
+import org.testng.annotations.Test;
+
+import java.util.Date;
+
+/**
+ * Created by adamk@wikia-inc.com on 9/25/15.
+ */
+@Test(groups = "ContentReview")
+public class ContentReviewTestMode extends NewTestTemplate {
+
+    @Test
+    @Execute(asUser = User.STAFF, onWikia = "openertest")
+    public void canEnterAndExitTestMode() {
+        String currentDatetime = new Date().toString();
+        String prependTestElement = "$('#WikiaArticle').prepend('<div id=\"content-review-selenium-test-element\">" + currentDatetime + "</div>');";
+
+        /**
+         * Step 0 - Prepare a clean state with auto-approving a clearing revision.
+         */
+        VisualEditModePageObject editPage = new VisualEditModePageObject(driver).open("mediawiki:common.js");
+        editPage
+                .getAceEditor()
+                .clearContent();
+        editPage.clickAutoApproveCheckbox().clickPublishButton();
+
+        /**
+         * Step 1 - Create a revision that appends the test element with the current timestamp
+         */
```

Comment:
You should get rid of comments in tests. should be easy enough to read code without comments to understand test nature